core/src/test/resources/com/google/googlejavaformat/java/testdata/B20128760.output

```diff
@@ -89,19 +89,13 @@ class B20128760 {
   {
     Stream<ItemKey> itemIdsStream =
         stream(members)
-            .flatMap(
-                m ->
-                    m.getFieldValues()
+            .flatMap(m -> m.getFieldValues()
```

Comment:
My colleagues (who use a lot of lambdas) vastly prefer having fewer line-breaks.  I'm presently trying to sell them on this tool, and I think they will have a hard time buying with the extra newline.  (Of course I could maintain a fork, but for a formatting tool, it's better to stay as close to upstream as possible, since it's a matter of industry-wide standardization).